### PR TITLE
Uploaded flowchart in Data-Playground Finnhub

### PR DESCRIPTION
**Pull request by Aatmaj1**
Adds flowchart image to Finnhub page.

Screenshot 2023-06-30 at 2 20 25 PM

### CODE REVIEW

File: content/en/data-playground/finnhub.md [hunk 1]
```diff
@@ -260,6 +260,9 @@ func main() {
 
 {{% /data-playground-wrapper %}}
 
+<!-- Flowchart for ensign and finnhub -->
+{{< widefigure src="/img/data-playground/finnhub_flowchart.png" alt="Finnhub integration with Ensign">}}
```
Comment (Aatmaj1) [inline content/en/data-playground/finnhub.md:264]: Please could you suggest a caption for flowchart ?

**Review by rebeccabilbro — COMMENTED** [review]
Thank you so much @Aatmaj1 -- this was a great idea and I'm glad you suggested it! Now we will have to make them for the other Data Playground sources 😀 

For the caption, how about something like "an example of a real-time MLOps pipeline for stock market data".

If you don't mind, I'd also like you to add a title for the image.  I think the title should be something like "Sample Real-time Stock Market MLOps Pipeline". 
*(Note: I tried to do it using markdown, but unfortunately it looks like there's a bug currently in the website that's preventing me from adding the title as markdown, so I'd like to add it directly to your image for the time being)* 

Could you make those two edits and push the changes up to this PR? You will probably need to do a `git pull` first, so that you get the latest version of your branch. Let me know if you need a hand!

**Comment by Aatmaj1** [comment]
> 

@rebeccabilbro I am working with Danielle to put the caption below the image, will update it soon

**Comment by Aatmaj1** [comment]
image

**Review by rebeccabilbro — APPROVED** [review]
Looks great! Thank you 😸 @Aatmaj1

**Pull request merged**